### ISSUE TITLE
server.py error

### ISSUE THREAD

**Issue by kiwironnie** [Synss/python-mbedtls#65]
**NOTE: Please use stackoverflow for support questions**.
This repository's issues are reserved for feature requests and bug reports.

## I am submitting a …

  - [x] bug report
  - [ ] feature request

## Description
in 2.3.0 executing ./server.py --dtls --debug 3  --port 9009 --address 149.28.170.96 --psk-store "CLI1=asecretkey","CLI2=Client_identity" gives:
_enable_debug_output(conf)
TypeError: Argument 'conf' has incorrect type (expected mbedtls._tls.MbedTLSConfiguration, got DTLSConfiguration)
There is no similar error in 2.2.0
### Steps to reproduce

1.
1.
1.

### Minimal demo of the problem

```python
>>> import mbedtls
...
```

**Comment by Synss** [comment]
You are correct. I will make a new release with a fix shortly. In the meantime, the programs would work as expected if you leave the `--debug` argument out.

**Comment by Synss** [comment]
2.3.1 is baking. That should be fixed in there.

Thank you for taking the time to report this issue to me!
